web/src/components/storage/SpacePolicyMenu.tsx

```diff
@@ -70,36 +65,24 @@ export default function SpacePolicyMenu({ modelDevice, device }) {
 
   const currentPolicy = driveUtils.spacePolicyEntry(modelDevice);
 
-  const PolicyItem = ({ policy }) => {
-    const isSelected = policy.id === currentPolicy.id;
-    // FIXME: use PF/Content with #component prop instead when migrating to PF6
-    const Name = () => (isSelected ? <b>{policy.label}</b> : policy.label);
-
-    return (
-      <MenuItem
-        itemId={policy.id}
-        isSelected={isSelected}
-        description={policy.description}
-        onClick={() => onSpacePolicyChange(policy.id)}
-      >
-        <Name />
-      </MenuItem>
-    );
-  };
-
   return (
-    <DeviceMenu
-      title={<span>{driveUtils.contentActionsDescription(modelDevice)}</span>}
-      activeItemId={currentPolicy.id}
-    >
-      <MenuGroup label={<SpacePolicySelectorIntro device={device} />}>
-        <MenuList>
-          <Divider />
-          {SPACE_POLICIES.map((policy) => (
-            <PolicyItem key={policy.id} policy={policy} />
-          ))}
-        </MenuList>
-      </MenuGroup>
-    </DeviceMenu>
+    <Flex gap={{ default: "gapXs" }}>
```

Comment:
We should sit down at some point and go through all these gaps used in the editors, so we can pick one and make sure all components use the same value wherever it makes sense.